static/js/place/place.ts

```diff
@@ -147,11 +147,7 @@ function renderPage(): void {
   const chartGeoJsonPromise = getGeoJsonData(dcid, placeType);
```

Comment:
can we return empty promises for these if not in USA and place type != country / state / county